### PR TITLE
generate docker image and push for development

### PR DESCRIPTION
**Pull request by kramasamy**

### CODE REVIEW

File: docker/base/scripts/generate-zookeeper-config.sh [hunk 1]
```diff
@@ -1,38 +1,157 @@
...
+    fi
+    if [ ! -f $ID_FILE ]; then
+        echo $MY_ID >> $ID_FILE
+    fi
+    echo "Created ZooKeeper data directories and set permissions in $ZK_DATA_DIR"
+}
+
+function create_log_props () {
+	rm -f $LOGGER_PROPS_FILE
+    echo "Creating ZooKeeper log4j configuration"
```
Comment (cckellogg) [inline docker/base/scripts/generate-zookeeper-config.sh]: alignment is off

File: docker/base/scripts/generate-zookeeper-config.sh [hunk 2]
```diff
@@ -1,38 +1,157 @@
...
+    echo "ENSEMBLE"
+    print_servers
+    echo "Enviorment validation successful"
+}
 
-    if [ "$HOSTNAME" == "$SERVER" ]; then
-        MY_ID=$IDX
-        echo "Current server id $MY_ID"
+function create_config() {
+	rm -f $ZK_CONFIG_FILE
```
Comment (cckellogg) [inline docker/base/scripts/generate-zookeeper-config.sh]: alignment looks off

File: docker/base/scripts/generate-zookeeper-config.sh [hunk 3]
```diff
@@ -1,38 +1,157 @@
...
+function print_servers() {
+	 for (( i=1; i<=$ZK_REPLICAS; i++ ))
+	do
+		echo "server.$i=$NAME-$((i-1)).$DOMAIN:$ZK_SERVER_PORT:$ZK_ELECTION_PORT"
+	done
+}
+
+function validate_env() {
+    echo "Validating enviornment"
+	if [ -z $ZK_REPLICAS ]; then
```
Comment (cckellogg) [inline docker/base/scripts/generate-zookeeper-config.sh]: alignment looks off

**Comment by jerrypeng** [comment]
+1

**Comment by kramasamy** [comment]
@cckellogg - kubernetes changes for the other yamls will be changed in the later PR.

**Pull request merged**